components/button/button.html

```diff
@@ -0,0 +1,263 @@
+      <header class="header flex--row">
+        <a href="/index.html">
+          <div class="header__logo-container flex--row">
+            <img src="/assests/zodiaclogo.png" alt="AtroUI logo" class="logo__img" />
+            <h2 class="header__logo">Astro UI</h2>
+          </div>
+        </a>
+        <nav class="header_nav flex--row">
+          <ul>
+            <li><a href="#">Documentation</a></li>
+            <li><a href="/components_page.html">Components</a></li>
+            <li><a href="https://github.com/ark2002/astro-ui">Github</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div class="btn__demo">
+      <p class="documentation">Solid Rectangular Buttons</p>
+    <div class="btn__basic">
+      <button class="btn btn-color--primary btn-font--primary">primary</button>
+      <button class="btn btn-color--secondary btn-font--primary">
+        secondary
+      </button>
+      <button class="btn btn-color--tertiary btn-font--primary">
+        tertiary
+      </button>
+      <button class="btn btn-color--primary btn-font--secondary">
+        primary
+      </button>
+      <button class="btn btn-color--secondary btn-font--secondary">
+        secondary
+      </button>
+      <button class="btn btn-color--tertiary btn-font--secondary">
+        tertiary
+      </button>
+    </div>
+    <p class="documentation">Solid Rounded Buttons</p>
+    <div class="btn__border--radius">
+      <button class="btn btn-color--primary btn-font--primary">primary</button>
+      <button class="btn btn-color--secondary btn-font--primary">
+        secondary
+      </button>
+      <button class="btn btn-color--tertiary btn-font--primary">
+        tertiary
+      </button>
+      <button class="btn btn-color--primary btn-font--secondary">
+        primary
+      </button>
+      <button class="btn btn-color--secondary btn-font--secondary">
+        secondary
+      </button>
+      <button class="btn btn-color--tertiary btn-font--secondary">
+        tertiary
+      </button>
+    </div>
+    <p class="documentation">Solid Button Bullets</p>
+    <div class="btn__bullets">
+      <button class="btn btn-color--primary btn-font--primary">primary</button>
+      <button class="btn btn-color--secondary btn-font--primary">
+        secondary
+      </button>
+      <button class="btn btn-color--tertiary btn-font--primary">
+        tertiary
+      </button>
+      <button class="btn btn-color--primary btn-font--secondary">
+        primary
+      </button>
+      <button class="btn btn-color--secondary btn-font--secondary">
+        secondary
+      </button>
+      <button class="btn btn-color--tertiary btn-font--secondary">
+        tertiary
+      </button>
+    </div>
+    <p class="documentation">Outline Rectangular Buttons</p>
+    <div class="btn__outline">
+      <button class="btn btn-color--primary btn-font--primary">primary</button>
+      <button class="btn btn-color--secondary btn-font--primary">
+        secondary
```

Comment:
i saw live preview, it just has buttons no documentation, please add that too before PRs. Mentioned by Kishan

Reply:
hey , currently lagging behind so my first priority currently is getting the components done first ill get done with the documentation as soon as i get done with these